library.wdl

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2018 Sequencing Analysis Support Core - Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import "readgroup.wdl" as readgroup
+import "sampleConfig.wdl" as sampleConfig
+
+workflow library {
+    Array[File] sampleConfigs
+    String sampleId
+    String libraryId
+
+    call sampleConfig.SampleConfig as readgroups {
+        input:
+            inputFiles = sampleConfigs,
+            sample = sampleId,
+            library = libraryId,
+            tsvOutputPath = "samples/" + sampleId + "/libs/" + libraryId + "/" + libraryId + ".config.tsv"
+    }
+
+    scatter (readgroupId in readgroups.keys) {
+        if (readgroupId != "") {
+            call readgroup.readgroup {
+                input:
+                    sampleConfigs = sampleConfigs,
+                    readgroupId = readgroupId,
+                    libraryId = libraryId,
+                    sampleId = sampleId
+            }
+        }
+    }
```

Comment:
Some comments where to put library jobs are missing

Reply:
done